go.mod

```
@@ -96,10 +97,10 @@ require (
 	github.com/ehazlett/simplelog v0.0.0-20200226020431-d374894e92a4
 	github.com/evanphx/json-patch v5.9.0+incompatible
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-git/go-git/v5 v5.12.0
+	github.com/go-git/go-git/v5 v5.13.0
```

Can we go for the latest path of that minor please? Otherwise, the version [v5.13.1](https://github.com/go-git/go-git/releases/tag/v5.13.1) fixes a regression from `v5.13.0`, so we probably want at least that patch.

```suggestion
	github.com/go-git/go-git/v5 v5.13.2
```